source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -29,10 +29,14 @@ Common::Redis::Client::PoolRequest* makeSingleServerRequest(
     const RouteSharedPtr& route, const std::string& command, const std::string& key,
     Common::Redis::RespValueConstSharedPtr incoming_request, ConnPool::PoolCallbacks& callbacks,
     Common::Redis::Client::Transaction& transaction) {
+  // If a transaction is active, clients_[0] is the primary connection to the cluster.
+  // The subsequent clients in the array are used for mirroring.
+  transaction.current_client_idx = 0;
```

Comment:
Haven't added the trailing underscore to all the usages.

Reply:
My mistake, the file didn't save because it had the wrong permissions. Fixed.